lib/vanagon/engine/local.rb

```diff
@@ -1,40 +1,38 @@
+require 'vanagon/engine/base'
 require 'vanagon/utilities'
 require 'vanagon/errors'
-require 'benchmark'
 
 class Vanagon
   class Engine
-    class Local
-      attr_accessor :target
+    class Local < Base
 
       def initialize(platform, target = nil)
         @platform = platform
         @target = "local machine"
         @name = 'local'
+        super
+        @required_attributes = Array.new
       end
 
       # Dispatches the command for execution
-      def dispatch(command)
-        puts Benchmark.measure { local_command(command, @workdir) }
-      end
-
-      # Steps needed to tear down or clean up the system after the build is
-      # complete
-      def teardown
-      end
-
-      # This method will take care of validation and target selection all at
-      # once as an easy shorthand to call from the driver
-      def startup(workdir)
-        @workdir = workdir
-        script = @platform.provisioning.join(' && ')
-        dispatch(script)
+      def dispatch(command, return_output = false)
+        # unset environment variables set by 'bundle exec build'
```

Comment:
Oh, I think this would work without requiring anything new:

```ruby
def dispatch(command, return_output = false)
  if defined?(Bundler)
    Bundler.with_clean_env do
      Vanagon::Utilities.local_command(command, return_command_output: return_output)
    end
  else
    Vanagon::Utilities.local_command(command, return_command_output: return_output)
  end
end
```

Testing this now.